### ISSUE TITLE
Synopsys ARC HSDK schematics

### ISSUE THREAD

**Issue by kubeqz** [foss-for-synopsys-dwc-arc-processors/ARC-Development-Systems-Forum#13]
Can you please provide schematic of the board ? Doesn't have to be full schematic, having at least I/Os wiring would be useful.

**Comment by iguryanov** [comment]
Hi, 

I/O pins are described in board user guide at https://github.com/foss-for-synopsys-dwc-arc-processors/ARC-Development-Systems-Forum/wiki/ARC-Development-Systems-Forum-Wiki-Home:
v1 board - https://github.com/foss-for-synopsys-dwc-arc-processors/ARC-Development-Systems-Forum/wiki/docs/ARC_HSDK_User_Guide.pdf 
v2 board - https://github.com/foss-for-synopsys-dwc-arc-processors/ARC-Development-Systems-Forum/wiki/docs/ARC_HSDK_4xD_User_Guide.pdf 

Would it have enough data? If not - please share your board version and ARC customer name and I'll check if we can produce better document. 

Regards,
Igor.

**Comment by kubeqz** [comment]
Yes, I saw these documents. I'm using v1 board. What I'm trying to achieve is to find GPIO mux settings for enabling the Mikrobus socket. After reading the user guide multiple times (and looking into [embarc_osp](https://github.com/foss-for-synopsys-dwc-arc-processors/embarc_osp/tree/master/board/hsdk/drivers/mux) source code ) I think I'm starting to understand how it's wired - however having a schematic that shows which GPIO pin is connected to which pin on the board would be a great help (but I guess this is still just _nice to have_).
